azext_iot/operations/dps.py

```diff
@@ -911,17 +913,25 @@ def _get_initial_twin(initial_twin_tags=None, initial_twin_properties=None):
 def _get_updated_inital_twin(
     enrollment_record, initial_twin_tags=None, initial_twin_properties=None
 ):
-    if initial_twin_properties != "" and not initial_twin_tags:
-        if hasattr(enrollment_record, "initial_twin"):
-            if hasattr(enrollment_record.initial_twin, "tags"):
-                initial_twin_tags = enrollment_record.initial_twin.tags
-    if initial_twin_properties != "" and not initial_twin_properties:
-        if hasattr(enrollment_record, "initial_twin"):
-            if hasattr(enrollment_record.initial_twin, "properties"):
-                if hasattr(enrollment_record.initial_twin.properties, "desired"):
-                    initial_twin_properties = (
-                        enrollment_record.initial_twin.properties.desired
-                    )
+    # in both cases, we want to grab the original tags and properties
+    # if the parameters are not provided. The user should be able to
+    # empty out tags/properties by passing in an empty string.
+    if (
```

Comment:
Are ITs passing with this change?

Reply:
I will rerun int tests for sanity

Reply:
<img width="1242" height="141" alt="image" src="https://github.com/user-attachments/assets/c3bce301-8112-4142-bd0d-e0b89a0b22c5" />